lib/travis/build/addons/sonarqube.rb

```diff
@@ -0,0 +1,46 @@
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Sonarqube < Base
+        SUPER_USER_SAFE = true
+        DEFAULT_SQ_HOST_URL = "https://nemo.sonarqube.org"
```

Comment:
@henryju or @dbmeneses Could you please update the default host and replace the URL by https://sonarqube.com? Thanks.